app/lists.rb

```diff
@@ -0,0 +1,73 @@
+module Listing
+	extend Discordrb::Commands::CommandContainer
+	
+	wing_list = Array.new
+	#The array that is wing_list is created.
+	
+#  def self.find_role(event, role)
+#    has_role = false
+#    event.server.roles.each do |server_role|
+#      next unless %w(botadmin wingcaptain).any?(server_role.name)
+#      has_role = true
+#    end
+#    return role
+#  end
+
+  def self.role?(event, role)
+    has_role = false
+    event.server.roles.each do |server_role|
+      next unless server_role.name == role
+      has_role = true
+      break
+    end
+    return has_role
+  end
+	
+  command :winglist, description: "Lists those who are requesting wing." do |event|
+  #Simple, return the array's contents. Also, admin lock this.
+    break unless event.user.role?(find_role(event, 'wingcaptain'))
+    #Check to see if they have the "wingcaptain role"
+    event << wing_list.take(wing_list.length)
```

Comment:
Its supposed to spit out the contents of the array in a list format. This might work.